docs/guides/tutorials/getting_started_v3_api/create_your_first_expectations.rst

```diff
@@ -0,0 +1,94 @@
+
+.. code-block:: bash
+
+    great_expectations --v3-api suite new
+
+You will see the following output:
+
+.. code-block:: bash
+
+    Using v3 (Batch Request) API
+    How would you like to create your Expectation Suite?
+        1. Manually, without interacting with a sample batch of data (default)
+        2. Interactively, with a sample batch of data
+        3. Automatically, using a profiler
+    : 3
+
+    A batch of data is required to edit the suite - let's help you to specify it.
+
+    Which data asset (accessible by data connector "taxi_data_example_data_connector") would you like to use?
+        1. yellow_tripdata_sample_2019-01.csv
+        2. yellow_tripdata_sample_2019-02.csv
+    : 1
+
+    Name the new Expectation Suite [yellow_tripdata_sample_2019-01.csv.warning]: taxi.demo
+
+    When you run this notebook, Great Expectations will store these expectations in a new Expectation Suite "taxi.demo" here:
+
+      <path_to_project>/great_expectations/expectations/taxi/demo.json
+
+    Would you like to proceed? [Y/n]: <press Enter>
+
+This will open up a **Jupyter notebook** that helps you create the new suite. Before diving into the notebook, let's first
+explain what we just did.
+
+**What just happened?**
+
+You may now wonder why we chose the first file in this step. Here's an explanation: Recall that our data directory
+contains two CSV files: `yellow_tripdata_sample_2019-01` and `yellow_tripdata_sample_2019-02`.
+
+* `yellow_tripdata_sample_2019-01` contains the January 2019 taxi data. Since we want to build an Expectation Suite based on what we know about our taxi data from the January 2019 data set, we want to use it for profiling.
+* `yellow_tripdata_sample_2019-02` contains the February 2019 data, which we consider the "new" data set that we want to validate before using it in production. We'll use it *later* when showing you how to validate data.
+
+Makes sense, right?
+
+After selecting the table, Great Expectations will open a Jupyter notebook which will take you through the next part of this workflow.
+
+.. warning::
+
+   Don't execute the Jupyter notebook cells just yet!
+
+
+Creating Expectations in Jupyter notebooks
+---------------------------------------------------------
+
+Notebooks are a simple way of interacting with the Great Expectations Python API. You could also just write all this in plain Python code, but for convenience, Great Expectations provides you some boilerplate code in notebooks.
+
+Since notebooks are often less permanent, creating Expectations in a notebook also helps reinforce that the source of truth about Expectations is the Expectation Suite, **not** the code that generates the Expectations.
+
+**Let's take a look through the notebook and see what's happening in each cell:**
+
+#. The first cell does several things: It imports all the relevant libraries, loads a Data Context, and creates what we call a Batch of your data and Expectation Suite.
```

Comment:
I like the idea of adding screenshots. I'll only do the first "screen" of the notebook though, since adding screenshots for the whole notebook would blow up the tutorial a lot. I'll do that for both versions of the tutorial.